airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/UsagePerConnectionTable.tsx

```diff
@@ -105,6 +114,152 @@ const UsagePerConnectionTable: React.FC<UsagePerConnectionTableProps> = ({ credi
+                isActive={sortBy === "destinationConnectionName"}
+                isAscending={sortOrder === SortOrderEnum.ASC}
+              >
+                <FormattedMessage id="credits.destination" />
+              </SortableTableHeader>
+            ),
+            meta: {
+              thClassName: classNames(styles.header, styles.light),
+              responsive: true,
+            },
+            cell: (props) => (
+              <NavLink
+                to={`/${RoutePaths.Workspaces}/${workspaceId}/${RoutePaths.Destination}/${props.row.original.destinationId}`}
+              >
+                <FlexContainer direction="row" alignItems="center">
+                  <ConnectorIcon icon={props.row.original.destinationIcon} />
+                  <Text size="sm">{props.cell.getValue()}</Text>
+                </FlexContainer>{" "}
+              </NavLink>
+            ),
+          }),
+          columnHelper.display({
+            id: "schedule",
+            header: () => (
+              <SortableTableHeader
+                onClick={() => {
+                  return null;
+                }}
+                isActive={false}
+                isAscending={false}
+              >
+                <FormattedMessage id="credits.schedule" />
+              </SortableTableHeader>
+            ),
+            cell: () => (
+              <FlexContainer className={styles.cell} alignItems="center">
+                <Text size="sm" className={styles.cellText}>
+                  Every 6 hours
+                </Text>
+              </FlexContainer>
+            ),
+            meta: {
+              thClassName: classNames(styles.header, styles.light),
+              responsive: true,
+            },
+          }),
+          columnHelper.accessor("creditsConsumedPercent", {
+            header: () => (
+              <SortableTableHeader
+                onClick={() => onSortClick("usage")}
+                isActive={sortBy === "usage"}
+                isAscending={sortOrder === SortOrderEnum.ASC}
+              >
+                <FormattedMessage id="credits.usage" />
+              </SortableTableHeader>
+            ),
+            meta: {
+              thClassName: classNames(styles.header, styles.light),
+              responsive: true,
+            },
+            cell: (props) => <UsageCell percent={props.cell.getValue()} />,
+          }),
+          columnHelper.accessor("creditsConsumed", {
+            header: "",
+            meta: {
+              thClassName: classNames(styles.header, styles.light),
+              responsive: true,
+            },
+            cell: (props) => (
+              <FlexContainer className={styles.cell} alignItems="center">
+                <Text className={styles.usageValue} size="lg">
+                  <FormattedNumber value={props.cell.getValue()} maximumFractionDigits={2} minimumFractionDigits={2} />
+                </Text>
+              </FlexContainer>
+            ),
+          }),
```

Comment:
Could we combine this column with the bar chart? I think semantically they belong together?